v3/glossary.md

```diff
@@ -0,0 +1,22 @@
+# Glossary
+
+* _channel_ : **TODO: find good description for channel**
+* _cloaking_ : method used to hides telehash traffic on the wire by randomizing all data sent
+* _Cipher Set (CS)_ : a collection of crypto algorithms with a given _cipher set id (CSID)_
+* _Cipher Set ID (CSID)_ : predefined hex number identifying a _cipher set (CS)_
+* _endpoint_ : a participant in the telehash network identified by a **(TODO: or multiple?)** _hashname_
+* _EVEN_ : the _endpoint_ with the numerically lower public key
+* _e3x_ : End-to-End Encrypted eXchange is a flexible encrypted exchange protocol
+* _exchange_ : **TODO: find good description for exchange**
+* _handshake_ : _message_ type used to establish an encrypted _session_ for _channels_
+* _hashname_ : an identifier for a participant of telehash, it is calculated from all public keys of the participants _cipher set (CS)_
+* _LOB_ : Lenght-Oobject-Binary encoding format that allows combining JSON and binary data
```

Comment:
This is also often called a "packet", or perhaps it's helpful to have packet be it's own glossary entry of "A LOB-encoded byte sequence sent between two endpoints"?